cfme/tests/infrastructure/test_vm_power_control.py

```diff
@@ -426,3 +448,52 @@ def test_delete(self, verify_vm_stopped, vm, vm_name, rest_api):
         wait_for(
             lambda: not rest_api.collections.vms.find_by(name=vm_name),
             num_sec=240, delay=5)
+
+    def test_edit(self, verify_vm_running, vm, vm_name, rest_api):
+        assert "edit" in vm.action
+        new_name = fauxfactory.gen_alphanumeric()
+        vm.action.edit(name=new_name)
+        wait_for(
```

Comment:
I added it for more strong verification condition, but waiting for vm with new_name is enough